profiles/arch/riscv/package.use.mask

```diff
@@ -200,7 +208,7 @@ net-irc/irssi socks5
 net-libs/gnutls doc test-full
 net-libs/ldns doc python
 net-libs/libpcap rdma
-net-misc/curl brotli curl_ssl_mbedtls curl_ssl_nss rtmp ssh nss mbedtls
+net-misc/curl brotli curl_ssl_mbedtls rtmp ssh nss mbedtls
```

Comment:
Please remove the _nss_ mask as well.

Reply:
...which in light of other changes here just committed to the tree means you should simply remove this line.

Reply:
Actually, I'm not sure if it's correct to remove all this USE flags. Simply remove this line might cause some issues in other places?

Reply:
Thank you! fixed it.